example/cajita_tutorial/10_spline/spline_example.cpp

```diff
@@ -17,6 +17,9 @@
 //---------------------------------------------------------------------------//
 // Spline example.
 //---------------------------------------------------------------------------//
+
+using namespace Cajita;
```

Comment:
Looks like you're not using this much. More clear to remove anyway

Reply:
Removed.